src/ui_basic/slider.h

```diff
@@ -121,9 +121,9 @@ class Slider : public Panel {
 	int32_t relative_move_;
 	int32_t big_step_;
 
-	bool highlighted_;  //  mouse over
-	bool pressed_;      //  the cursor is pressed
-	bool enabled_;      //  enabled widget
+	bool highlighted_{false};  //  mouse over
+	bool pressed_{false};      //  the cursor is pressed
```

Comment:
these 2 bools are initilized here and in the constructor together with other vlaues like big step please recheck